File: PhysicalTherapy/PhyscialTherapy/Client/src/app/models/MessageLog.ts
import { Patient } from "./Patient";
import { Routine } from "./Routine";
import { Therapist } from "./Therapist";

export interface MessageLog {
    insertDate : Date,
    message : string,
    messageLogId : number,
    patient : Patient,
    patientId : number,
    routine : Routine,
    routineId : number,
    therapist : Therapist,
    therapistId : number

}

File: PhysicalTherapy/PhysicalTherapy/Models/MessageLog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PhysicalTherapy.Models
{
    public class MessageLog
    {
        public DateTime InsertDate { get; set; }

        public string Message { get; set; }

        public int MessageLogId { get; set; }

        public Patient Patient { get; set; }

        public int? PatientId { get; set; }

        public Routine Routine { get; set; }

        public int? RoutineId { get; set; }

        public Therapist Therapist { get; set; }

        public int? TherapistId { get; set; }
    }
}
